src/routes/sessions/[uuid]/.Requester/.Requester.svelte

```diff
@@ -3,46 +3,45 @@
   import Notification from '$components/Notification/Notification.svelte';
   import { NOTIFICATION_TYPES, NOTIFICATION_VALUES } from '$components/Notification/enums';
   import { DEVICE_CALL_QUEUE_SETUP_GUIDE } from '$lib/constants';
-
+  import defaultSupportImg from '$lib/static/img/customer-support.png';
```

Comment:
For static assets, the import name should follow the file name, with changed casing and removed extension.